src/main/java/qna/domain/Answer.java

```diff
@@ -9,21 +12,24 @@
 
 @Entity
 @Table
-public class Answer {
-    @Id
-    @GeneratedValue(strategy = GenerationType.IDENTITY)
-    private Long id;
-    @Column
-    private Long writerId;
-    @Column
-    private Long questionId;
+@NoArgsConstructor
```

Comment:
외부에서는 생성자를 통해 `Entity`를 생성할 일이 없으므로 `access level`을 조정하시는걸 추천 드립니다 😄 
```suggestion
@NoArgsConstructor(access = AccessLevel.PROTECTED)
```